cnd/src/network.rs

```diff
@@ -513,13 +513,15 @@ impl libp2p::swarm::NetworkBehaviourEventProcess<::comit::network::comit::Behavi
     }
 }
 
-impl libp2p::swarm::NetworkBehaviourEventProcess<announce::BehaviourOutEvent> for ComitNode {
-    fn inject_event(&mut self, event: announce::BehaviourOutEvent) {
+impl libp2p::swarm::NetworkBehaviourEventProcess<announce::BehaviourOutEvent<LocalSwapId>>
+    for ComitNode
+{
+    fn inject_event(&mut self, event: announce::BehaviourOutEvent<LocalSwapId>) {
         match event {
             announce::BehaviourOutEvent::Confirmed {
                 peer,
                 shared_swap_id,
-                local_swap_id,
+                context: local_swap_id,
```

Comment:
Does this change the way the message is formed over the network?

Reply:
The context is never sent over the network! :)

This is just an identifier that is stored locally and re-emitted with the event so the caller can treat the event accordingly.